Shop.BLL/Exceptions/NotFoundExceptions/GameNotFoundException.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Shop.BLL.Exceptions.NotFoundExceptions
+{
+    public class GameNotFoundException : NotFoundException
+    {
+        public GameNotFoundException(Guid id) : base($"Game with id: {id} not found.") 
```

Comment:
it is better to create a NotFoundException and pass a string message to the constructor and call it in the service and send a specific message already

Reply:
Done